src/data_provider/src/sysInfoWin.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Wazuh SysInfo
- * Copyright (C) 2015, Wazuh Inc.
+ * Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
According to this PR https://github.com/wazuh/wazuh/pull/11780/, current year is not necessary anymore
```suggestion
 * Copyright (C) 2015, Wazuh Inc.
```